src/routes/ChatFinished/container.js

```diff
@@ -32,7 +32,7 @@ export const ChatFinishedConnector = ({ ref, ...props }) => (
 				{...props}
 				title={I18n.t('Chat Finished')}
 				color={color}
-				greeting={greeting || I18n.t('Thanks to talk with us')}
+				greeting={greeting || I18n.t('Thanks for talking with us')}
 				message={I18n.t('If you have any other question, just press the button below to start a new chat.')}
```

Comment:
Sorry I didn't notice this before... Question should have an "s" at the end of it.
```message={I18n.t('If you have any other questions, just press the button below to start a new chat.')}```

Reply:
@wreiske, fixed!
Thanks.